src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/GeneralPartnerController.java

```diff
@@ -43,6 +45,25 @@ public GeneralPartnerController(GeneralPartnerService generalPartnerService) {
         this.generalPartnerService = generalPartnerService;
     }
 
+    @GetMapping("/{" + URL_PARAM_GENERAL_PARTNER_ID + "}")
+    public ResponseEntity<GeneralPartnerDto> getGeneralPartner(@RequestAttribute(TRANSACTION_KEY) Transaction transaction,
+                                                               @PathVariable(URL_PARAM_GENERAL_PARTNER_ID) String submissionId,
+                                                               @RequestHeader(value = ERIC_REQUEST_ID_KEY) String requestId) {
+        String transactionId = transaction.getId();
+        HashMap<String, Object> logMap = new HashMap<>();
+        logMap.put(URL_PARAM_TRANSACTION_ID, transactionId);
+        logMap.put(URL_PARAM_SUBMISSION_ID, submissionId);
+
+        try {
+            ApiLogger.infoContext(requestId, "Retrieving a general partner", logMap);
```

Comment:
can you add the general partner id in the log message, if we get errors, we will know what record it was trying to access

Reply:
Added this to the log message.